waspc/data/Generator/templates/react-app/src/auth/logout.js

```diff
@@ -1,8 +1,8 @@
-import { clearLocalStorage } from '../api.js'
+import { cleanupStorage } from '../api.js'
 import { invalidateAndRemoveQueries } from '../operations/resources'
 
 export default async function logout() {
-  clearLocalStorage()
+  cleanupStorage()
```

Comment:
What do we really want to do here?
We want to clear any of the local storage / cache connected to the user? But also connected to the Wasp? What would be the full sentence that very precisely defines what we are doing here? I think this is important to nail down, and then we can make sure that function we call here reflects that and is a correct abstraction for it.
So let's nail down the intention here first, and then based on that we might even want to take that functino out of api.js -> that is what it looks like to me right now.

Reply:
I've renamed it to be `removeLocalUserData`